test/test_layout_blspec.py

```diff
@@ -35,21 +35,20 @@ def setUp(self) -> None:
         os.environ['XDG_CONFIG_HOME'] = '/dev/null'
 
     def create_layout(self,
-                      efi_subdir: bool = False,
```

Comment:
FWICS there isn't any test for ESP being mounted in `/boot/EFI` now.

Reply:
Yeah the way I changed the tests is wrong, I'm gonna have to go back and rethink that.

I somehow interpreted wrong what the `test_find_modules_EFI` test was supposed to be doing.

Reply:
Fixed, now I simply adjust the test spec while touching the actual test functions only minimally. This makes a lot more sense then the mess I made before.